OpenWormData/scripts/insert_worm.py

```diff
@@ -158,9 +161,11 @@ def upload_neurons():
 def get_altun_evidence():
     return parse_bibtex_into_evidence('../aux_data/bibtex_files/altun2009.bib')
 
+
```

Comment:
Changes for pep8 compliance are fine, but please don't introduce newlines unnecessarily.